AGM-1977 Update admins documentation (#439)
AdguardTeam/KnowledgeBase: docs/adguard-for-windows/admins-documentation.md

If you want to install AdGuard on a Windows 7 computer, make sure that it has .NET 4 Client Profile installed: https://www.microsoft.com/en-us/download/details.aspx?id=24872

:::note

If you have installed AdGuard for Windows from an MSI file, AdGuard will not be updated automatically. To allow manual updates:

> * in versions released after v7.16, delete the `ForbidCheckUpdates` parameter or set its value to `NO` (case-insensitive) in the registry under `HKEY_LOCAL_MACHINE\SOFTWARE\Adguard`.  

Reviewer: ```suggestion
* in versions released after v7.16, delete the `ForbidCheckUpdates` parameter or set its value to `NO` (case-insensitive) in the registry under `HKEY_LOCAL_MACHINE\SOFTWARE\Adguard`.
```

Reviewer: проверяй во вскоде такие штуки сразу

> * in v7.16 and earlier versions, set to `YES` (case-insensitive) the `AllowCheckUpdates` parameter for the `HKEY_LOCAL_MACHINE\SOFTWARE\Adguard` key on x86 OS or `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\Adguard` key on x64 OS. Any other value or no value for this parameter will disable automatic updates.

Reviewer: ```suggestion
* in v7.16 and earlier versions, set the `AllowCheckUpdates` parameter to `YES` (case-insensitive) for the `HKEY_LOCAL_MACHINE\SOFTWARE\Adguard` key on x86 OS or `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\Adguard` key on x64 OS. Any other value or no value for this parameter will disable automatic updates.
```

:::info

> When installing AdGuard for Windows from MSI, AdGuard is not automatically updated. If you want to allow updates for the user (which is not recommended, because it prevents centralized updates), you must either delete the `ForbidCheckUpdates` parameter or set its value to `NO` (not case-sensitive) in the registry under `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\Adguard`. Doing so will permit updates.

Reviewer 2: в процессе комментирования осознал, что надо проверить кой-чего. Отпишу тут по результату финальный вариант, т.е. пока ничего менять не надо

> :::note

Reviewer 2: ещё действительно в формате [примечания](
https://files.slack.com/files-tmb/T0B41S97U-F06NV25F41Y-3539141e8c/image_720.png) надо добавить, что 
```
before 7.16 was released in order to allow updates for the user, set the value of the `AllowCheckUpdates` parameter for the `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\Adguard` key to `YES` (case insensitive). In this case automatic updates will be allowed, any other value or no value for this parameter disables automatic updates.
```

:::info

> When installing AdGuard for Windows from MSI, AdGuard is not automatically updated. If you want to allow updates for the user (which is not recommended, because it prevents centralized updates), you must either delete the `ForbidCheckUpdates` parameter or set its value to `NO` (not case-sensitive) in the registry under `HKEY_LOCAL_MACHINE\SOFTWARE\Adguard`. Doing so will permit updates.

Reviewer: ```suggestion
When installing AdGuard for Windows from MSI, AdGuard is not automatically updated. If you want to allow updates for the user (which is not recommended, because it prevents centralized updates), you must either delete the `ForbidCheckUpdates` parameter or set its value to `NO` (case-insensitive) in the registry under `HKEY_LOCAL_MACHINE\SOFTWARE\Adguard`. This will allow updates.
```

Reviewer: что-то я не улавливаю: «При установке AdGuard для Windows из MSI AdGuard не обновляется автоматически. Если вы хотите разрешить обновления для пользователя (что не рекомендуется, поскольку это препятствует централизованным обновлениям), вы должны...». в первом предложении мы говорим, что адгард не обновляется автоматически (системой), а во втором говорим: «если вы хотите разрешить обновления для пользователя (что препятствует централизованным обновлениям — отключенным)... непонятно

Reviewer 2: в общем тут тема такая. Если пользователь установил наш exe, то автоматически приходят обновления и всё хорошо. Если же это гиковская установка через msi (например, на предприятии, чтобы сделать централизованно), то это самое автоматическое обвновление отключено (по многим причинам). 

>Если вы хотите разрешить обновления для пользователя (что не рекомендуется, поскольку это препятствует централизованным обновлениям)

Но мы должны предложить вариант пользователю наплевать на нас и всё равно обновляться (на свой страх и риск - там на самом деле есть тезнические проблемы)

>что препятствует централизованным обновлениям

в том смысле, что эти самые централизованные обновления будут централизованно идти от местного сисадмина

Reviewer: тогда предлагаю так переписать:

Если вы установили AdGuard для Windows из файла MSI, AdGuard не будет обновляться автоматически. Чтобы разрешить обновления для пользователя, вы должны удалить параметр `ForbidCheckUpdates` или установить его значение `NO` (регистр не важен) в реестре под `HKEY_LOCAL_MACHINE\SOFTWARE\Adguard`. Но обратите внимание, что такая установка приложения и ручные обновления влекут за собой технические проблемы и не рекомендуются.

:::

:::note

> In v7.16 and earlier versions of AdGuard for Windows, to allow updates for the user, the `AllowCheckUpdates` parameter for the `HKEY_LOCAL_MACHINE\SOFTWARE\Adguard` key key on x86 OS or `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\Adguard` key on x64 OS must be set to `YES` (case insensitive). In this case automatic updates will be allowed, any other value or no value for this parameter disables automatic updates.

Reviewer: ```suggestion
In v7.16 and earlier versions of AdGuard for Windows, to allow updates for the user, the `AllowCheckUpdates` parameter for the `HKEY_LOCAL_MACHINE\SOFTWARE\Adguard` key key on x86 OS or `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\Adguard` key on x64 OS must be set to `YES` (case-insensitive). Any other value or no value for this parameter will disable automatic updates.
```
